Sources/DistributedSystem/DistributedSystem.swift

```diff
@@ -30,6 +29,14 @@ extension Channel {
     }
 }
 
+public final class Box<T> {
```

Comment:
Needs to be public?

Reply:
Yes, it there is a public member variable of that type...